src/test/java/com/ericsson/ei/queryservice/test/QueryServiceRESTAPITest.java

```diff
@@ -81,7 +79,7 @@
     private QueryAggregatedObjectControllerImpl aggregatedObjectController;
 
     @MockBean
-    private QueryMissedNotificationControllerImpl missedNotificationController;
+    private FailedNotificationsControllerImpl failedNotificationController;
```

Comment:
the class name is imported correctly, but the variable name is still in singular and not plural. "failedNotificationController" --> "failedNotificationsController"

Reply:
done